userspace/libsinsp/user.cpp

```diff
@@ -592,17 +592,9 @@ bool sinsp_usergroup_manager::user_to_sinsp_event(const scap_userinfo *user, sin
 
 	scap_evt* scapevt = evt->m_pevt;
 
-	if(m_inspector->m_lastevent_ts == 0)
-	{
-		// This can happen at startup when containers are
-		// being created as a part of the initial process
-		// scan.
-		scapevt->ts = sinsp_utils::get_current_time_ns();
-	}
-	else
-	{
-		scapevt->ts = m_inspector->m_lastevent_ts;
-	}
+	scapevt->ts = (m_inspector->m_lastevent_ts == 0)
```

Comment:
this code is repeated many times
```cpp
 (m_inspector->m_lastevent_ts == 0)
		? sinsp_utils::get_current_time_ns()
		: m_inspector->m_lastevent_ts;
```
maybe it could become an `inspector` method, WDYT?